forch/simple_auth_state_machine.py

```diff
@@ -13,83 +13,89 @@ class AuthStateMachine():
     UNAUTH = "Unauthorized"
     REQUEST = "RADIUS Request"
     ACCEPT = "Authorized"
-    MAX_RADIUS_RETRIES = 5
+    MAX_RADIUS_BACKOFF = 5
     QUERY_STATE_TIMEOUT = 10
-    REJECT_STATE_TIMEOUT = 20
-    AUTH_STATE_TIMEOUT = 20
+    REJECT_STATE_TIMEOUT = 300
+    AUTH_STATE_TIMEOUT = 3600
 
-    def __init__(self, src_mac, port_id, radius_query_callback, auth_callback):
+    # pylint: disable=too-many-arguments
+    def __init__(self, src_mac, port_id, auth_config, radius_query_callback, auth_callback):
         self.src_mac = src_mac
         self.port_id = port_id
-        self.auth_callback = auth_callback
-        self.radius_query_callback = radius_query_callback
-        self.current_state = None
+        self._auth_callback = auth_callback
+        self._radius_query_callback = radius_query_callback
+        self._current_state = None
         self._retry_backoff = 0
-        self.current_timeout = 0
-        self.transition_lock = Lock()
+        self._current_timeout = 0
+        self._max_radius_backoff = auth_config.get('max_radius_backoff', self.MAX_RADIUS_BACKOFF)
+        self._query_state_timeout = auth_config.get('query_state_timeout', self.QUERY_STATE_TIMEOUT)
```

Comment:
configs should be suffixed with units (_ms? _sec?) -- also drop "state" -- e.g., "query_timeout_ms"

Reply:
Does this need to be changed in the config fields as well or only the variables?

Reply:
Done